google-home-community.groovy

```diff
@@ -2334,6 +2375,19 @@ private traitFromSettings_OpenClose(traitName) {
     return openCloseTrait
 }
 
+@SuppressWarnings('UnusedPrivateMethod')
+private traitFromSettings_CameraStream(traitName) {
+    // NOTE: as of 2020-08-01 Google does not support any protocols other than Chromecast.  Protocols below are unused defaults, and StreamToChromecast must be true.
+    return [
+        cameraStreamAttribute: settings."${traitName}.cameraStreamAttribute",
+        commands:        [],
+        params: [
+            StreamToChromecast: true,
+            SupportedStreamProtocols: ["progressive_mp4", "hls", "dash", "smooth_stream"]   
+        ]
```

Comment:
I don't see where this `params` map is used.  Is it needed?

Reply:
Agreed, will be removed.